flow/packet-headers/ipv6_routing.yaml

```diff
@@ -0,0 +1,525 @@
+            description: 16-bit field used to tag a packet as part of a class or group,
+              enabling shared properties or policy treatment. If not used, it MUST
+              be set to zero.
+            format: integer
+            length: 16
+            default: 0
+            features:
+              - count
+          x-field-uid: 6
+        segment_list:
+          description: 128-bit IPv6 addresses representing the nth segment in the
+            Segment List. The Segment List is encoded starting from the last segment
+            of the SR Policy. That is, the first element of the Segment List (Segment List[0])
+            contains the last segment of the SR Policy, the second element contains
+            the penultimate segment of the SR Policy, and so on.
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/Flow.Ipv6SegmentRouting.Segment'
+          x-field-uid: 7
+        tlvs:
+          $ref: '#/components/schemas/Flow.Ipv6SegmentRouting.Tlvs'
+          x-field-uid: 8
+    Flow.Ipv6SegmentRouting.Segment:
+      type: object
+      properties:
+        segment:
+          x-field-pattern:
+            description: A 128-bit IPv6 address segment.
+            format: ipv6
+            default: ::0
+            features:
+              - count
+          x-field-uid: 1
+    Flow.Ipv6SegmentRouting.Tlvs:
+      description: A list of optional Type-Length-Value (TLV) objects that carry
+        metadata for segment processing. TLVs are only present if the Hdr Ext
+        Len is greater than what is required for the fixed header and the segment
+        list.
+      type: object
+      properties:
+        ingress:
+          $ref: '#/components/schemas/Flow.Ipv6SegmentRouting.TlvIngressNode'
+          x-field-uid: 1
+        egress:
+          $ref: '#/components/schemas/Flow.Ipv6SegmentRouting.TlvEgressNode'
+          x-field-uid: 2
+        opaque:
+          $ref: '#/components/schemas/Flow.Ipv6SegmentRouting.TlvOpaqueContainer'
+          x-field-uid: 3
+        padding:
+          $ref: '#/components/schemas/Flow.Ipv6SegmentRouting.TlvPad'
+          x-field-uid: 4
+        hmac:
+          $ref: '#/components/schemas/Flow.Ipv6SegmentRouting.TlvHmac'
+          x-field-uid: 5
+        path_trace:
+          $ref: '#/components/schemas/Flow.Ipv6SegmentRouting.TlvPathTrace'
+          x-field-uid: 6
+    Flow.Ipv6SegmentRouting.Flags:
+      description: An 8-bit field containing flags. While RFC 8754 reserves all
+        bits as unused, earlier drafts defined specific flags for behavior such
+        as OAM, HMAC, and FRR protection.
+      type: object
+      properties:
+        u1_flag:
+          x-field-pattern:
+            description: Unused bit. Should be set to 0.
+            format: integer
+            length: 1
+            default: 0
+            features:
+              - count
+          x-field-uid: 1
+        p_flag:
```

Comment:
I think, go snappi code will good if change the properly.
Flags().SetProtected(true)
Flags().SetOAM(true)
Flags().SetAlert(true) etc...